### DISCUSSION TITLE
Multireddit : delete a sub as you can add one

### DISCUSSION THREAD

**Discussion by skal-dB** [Docile-Alligator/Infinity-For-Reddit#1324] [Ideas]
It's possible to add a subreddit to a multireddit from the 3-dots button, it would be convenient to have be able to make it the other way also.

Dunno if it's possible thou.

**Comment by MahouShoujoMivutilde** [comment]
_I agree completely!_

### Why this is a much-needed feature

When you have a lot of subscriptions on various topics multireddits are essential to group things up and make it more manageable.   

At the moment **to add a subreddit** to a multireddit it takes **3 clicks**, assuming the desired subreddit is already open: 

* three dots → add to multireddit →  select multireddit.

**This is reasonable and convenient.**

However, **to remove** a subreddit from a multireddit you need to: 

* **exit the subreddit** (inconvenient) → draw the side menu from the left → open "multireddit" → long press on multireddit _you need_ → pick "edit multireddit" → press "select subreddits and users" → find the subreddit you want to remove (no search and no icons) and press trash icon → press checkmark to confirm  → press checkmark to confirm **again** ... And you're done! 

Now you might want to continue where you left... Except you left the subreddit page.

So **9 clicks** and **you lose where you were**.

### Proposal 

Have a **"Remove subreddit from multireddit" action** right next to "Add subreddit to multireddit".

The action should open a list of users multireddits (ideally, only the ones with the subreddit) for user to select. After selection is complete the subreddit should be removed from the selected multireddits (some people _might_ have the same sub in multiple multireddits) without further user input.

_This will greatly improve user experience with multireddits._
